### ISSUE TITLE
WCAG 2.1 Understanding doc does not have SC numbers

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#370]
I find it challenging that the understanding document does not have numbers for the SC.  While most may not rely on these numbers it makes it hard to match up items from WCAG 2.1 to the understanding docs.  I find the numbers helpful for matching between the two documents.
https://www.w3.org/WAI/WCAG21/Understanding/.   People opposed to the numbers will say that people don't have them memorized.  I'm not suggesting that.  I'm saying that they are helpful when working between documents as the number is shorter to recall than the long phrase.

**Comment by Ryladog** [comment]
+1 to Jon

**Comment by alastc** [comment]
The [previous version](https://www.w3.org/TR/UNDERSTANDING-WCAG20/) was very verbose, it had "Understanding Success Criterion " at the start of every link, I don't think we have to go back to that.

If we change that table of contents (TOC), presumably something like the [guidelines TOC](https://www.w3.org/TR/WCAG21/#table-of-contents) would be suitable?

@michael-n-cooper the source for that is toc.html, but is that a generated file or a simple include? My XSL is rusty, to say the least. (And where else is it used?)
